src/plugins/Wallet/wallet.ts

```diff
@@ -79,6 +84,56 @@ export async function getManagedWallets(): Promise<{
     return { wallets: await makeWallets(), tokens: makeTokens() }
 }
 
+export async function updateExoticWalletsFromSource(
```

Comment:
Plz divide the function into `updateExoticWalletFromSource` and `updateExoticWalletsFromSource`. The former one for the amniotic purpose.

Reply:
I don't understand